packages/insight/app/src/pages/broadcast-tx/broadcast-tx.ts

```diff
@@ -4,9 +4,10 @@ import { ToastController } from 'ionic-angular';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Http } from '@angular/http';
 import { ApiProvider } from '../../providers/api/api';
+import { NavParams } from 'ionic-angular/navigation/nav-params';
 
 @IonicPage({
-  segment: ':selectedCurrency/broadcast-tx'
+  segment: ':selectedCurrency/:selectedNetwork/broadcast-tx'
```

Comment:
Do you want to change this to `:chain/:network` to be consistent with other segment definitions?